cotainr/util.py

```diff
@@ -133,7 +133,14 @@ def stream_subprocess(*, args, log_dispatcher=None, **kwargs):
         stderr="".join(captured_stderr),
     )
 
-    completed_process.check_returncode()
+    try:
+        completed_process.check_returncode()
+    except subprocess.CalledProcessError as err:
+        print(f"A subprocess command {err.cmd} " +
+              f"failed with returncode {err.returncode}\n",
+              f"Returned error:\n {err.stderr}"
+              f"Returned output:\n {err.stdout}")
+        raise
```

Comment:
I am a bit unsure about how this interacts with `logging.exception(...)`, but we should probably still raise e.g. a `RuntimeError("...") from err` to make the context of the exception more clear. However, the most important part here is that the error message is easy to decipher for the user.